---
source: crates/uitest/tests/ty.rs
assertion_line: 20
expression: diags
input_file: fixtures/ty/def/associated_types.fe
---
error[2-0002]: `Output` is not found
  ┌─ associated_types.fe:4:39
  │
4 │     fn add(self, _ other: T) -> Self::Output // TODO: fix 'not found' error
  │                                       ^^^^^^ `Output` is not found

error[6-0008]: method return type mismatch
   ┌─ associated_types.fe:14:35
   │
14 │     fn add(self, _ other: u16) -> Val {
   │                                   ^^^ expected `invalid(Other)` type, but the given type is `invalid(Other)`
